### DISCUSSION TITLE
Installation of Instant Clue on Mac

### DISCUSSION THREAD

**Discussion by angkoomin** [hnolCol/instantclue#14] [Q&A]
Hi there,

I have just seen videos about Instant Clue and it looks fascinating. However, I am having difficulties installing it on a mac. I was wondering if someone can guide me with the installation?

Thanks

**Comment by hnolCol** [comment]
Hi, 

please excuse my late reply. InstantClue has been given a facelifting and moved to the PyQt5 framework. 

You should now be able to just download - extract and start from [here](https://github.com/hnolCol/instantclue/releases). Make sure to allow InstantClue to open as we are "unidentified developers" on MacOS. 

If things go wrong, please create an issue. 

Best wishes and if you have any feature request, please post it instantaneously ( :) ) in the discussion forum

Hendrik

**Reply by angkoomin** [reply]
Is there a readme file for the installation. I am not sure if has been installed completely on my Mac. I don't seem to be able to load my files (either as csv, txt or xlsx files).

**Reply by hnolCol** [reply]
Hi, 

does the software start? Then you "installed" it properly. It should work automatically after extracting the zip file (usually automatically done on Mac). If it does not open, try moving it to the desktop or the Application and set the security settings to allow for software that is designed by undefined developers. (In the Download folder, there seem to be some problems with a path).

If the software starts, does it show an error while reading your file in the top right corner of your screen? 
Tutorials will be added shorty
